virttest/libvirt_xml/nwfilter_binding.py

```diff
@@ -0,0 +1,148 @@
+
+    def __init__(self, virsh_instance=base.virsh):
+        accessors.XMLElementNest("owner", self,
+                                 parent_xpath='/',
+                                 tag_name='owner',
+                                 subclass=self.Owner,
+                                 subclass_dargs={
+                                     'virsh_instance': virsh_instance})
+        accessors.XMLAttribute('portdev', self,
+                               parent_xpath='/',
+                               tag_name='portdev',
+                               attribute='name')
+        accessors.XMLAttribute('mac_address', self,
+                               parent_xpath='/',
+                               tag_name='mac',
+                               attribute='address')
+        accessors.XMLElementNest("filterref", self,
+                                 parent_xpath='/',
+                                 tag_name='filterref',
+                                 subclass=self.Filterref,
+                                 subclass_dargs={
+                                     'virsh_instance': virsh_instance})
+        super(NwfilterBinding, self).__init__(virsh_instance=virsh_instance)
+
+        self.xml = u'<filterbinding></filterbinding>'
+
+    def new_filterref(self, **dargs):
+        """
+        Return a new interafce filterref instance from dargs
+        """
+        new_one = self.Filterref(virsh_instance=self.virsh)
+        for key, value in list(dargs.items()):
+            setattr(new_one, key, value)
+        return new_one
+
+    def new_owner(self, name, uuid):
+        """
+        Return a new interafce Owner instance from name and uuid
+        """
+        new_one = self.Owner(virsh_instance=self.virsh)
+        new_one.name = name
+        new_one.uuid = uuid
+        return new_one
+
+    class Owner(base.LibvirtXMLBase):
+        """
+        Interface Owner xml class
+
+        Properties:
+        name: string, name of the vm
+        uuid: string, domain uuid of vm
+        """
+        __slots__ = ("name", "uuid")
+
+        def __init__(self, virsh_instance=base.virsh):
+            accessors.XMLElementText('name', self,
+                                     parent_xpath='/',
+                                     tag_name='name')
+            accessors.XMLElementText('uuid', self,
+                                     parent_xpath='/',
+                                     tag_name='uuid')
+            super(self.__class__, self).__init__(virsh_instance=virsh_instance)
+            self.xml = u'<owner></owner>'
+
+    class Filterref(base.LibvirtXMLBase):
+        """
+        Interface filterref xml class.
+
+        Properties:
+
+        name:
+        string. filter name
+        parameters:
+        list. parameters element dict list
+        """
+        __slots__ = ("name", "parameters")
+
+        def __init__(self, virsh_instance=base.virsh):
+            accessors.XMLAttribute(property_name="name",
```

Comment:
Better keep property_name same with attribute

Reply:
thks, i change the interface.py to make it scalable